spec/features/collections/loss_report_spec.rb

```diff
@@ -0,0 +1,40 @@
+
+describe 'Displaying the comparison report in browser' do
+
+  context 'when accessing the comparison report' do
+
+    before do
+      login
+    end
+
+    context 'when displaying json' do
+      it 'properly displays the echo json report' do
+        visit loss_report_collections_path(echo_id, format:'json')
+        expect(page).to have_content('application/echo')
+      end
+      it 'properly displays the iso json report' do
+        visit loss_report_collections_path(iso_id, format:'json')
+        expect(page).to have_content('application/iso')
```

Comment:
So it is good that you have the helper tests to test those methods, but when we are doing feature tests/integration tests, we want to test to make sure that everything is working together. So I think that just checking that it has the right format is not sufficient, you should also test the report result as you are doing in the helper tests.